website/addons/mendeley/model.py

```diff
@@ -0,0 +1,352 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+import mendeley
+from modularodm import fields
+
+from framework.exceptions import PermissionsError
+
+from website.addons.base import AddonNodeSettingsBase
+from website.addons.base import AddonUserSettingsBase
+from website.oauth.models import ExternalProvider
+from website.util import web_url_for
+
+from website.addons.citations.utils import serialize_account, serialize_folder
+
+from . import settings
+from .api import APISession
+
+class AddonMendeleyUserSettings(AddonUserSettingsBase):
+    oauth_grants = fields.DictionaryField()
```

Comment:
Can you document the expected format for this field? Example: https://github.com/lyndsysimon/osf.io/blob/0cdf9152e0993ae63659b2835f4f612219e02ffe/website/project/model.py#L170-175